olive/engine/engine.py

```diff
@@ -224,8 +243,68 @@ def run(
             json.dump(results, open(results_path, "w"), indent=4)
             return results
 
+        # TODO by myguo: replace the following for loop using the accelerator and excution provider list when adding
+        # accelerator support
+        footprints = []
```

Comment:
How about use `dict[accelerator_name, footprints]'?

Reply:
will make the change when I add accelerator.